### PR TITLE
Choose destination for Parameter Encoding

### PR DESCRIPTION
**Pull request by asteiman**
Issue: https://github.com/jdisho/TinyNetworking/issues/6

### CODE REVIEW

**Comment by jdisho** [comment]
Haven't forgotten about this. I will get back hopefully during the weekend ☺️

File: Sources/TinyNetworking/URLEncoding.swift [hunk 1]
```diff
@@ -37,13 +37,20 @@ public struct URLEncoding {
...
+        case urlQuery
+        case httpBody
+    }
 
     public let arrayEncoding: ArrayEncoding
     public let boolEncoding: BoolEncoding
+    public let destination: Destination
 
-    public init(arrayEncoding: ArrayEncoding = .brackets, boolEncoding: BoolEncoding = .literal) {
+    public init(destination: Destination = .urlQuery, arrayEncoding: ArrayEncoding = .brackets, boolEncoding: BoolEncoding = .literal) {
```
Comment (jdisho) [inline Sources/TinyNetworking/URLEncoding.swift]: Does it make sense to make `destination` optional? 
Optional would mean that if it's `nil`, the destination would be inferred from the `HTTP` method. WDYT?

Comment (asteiman) [reply Sources/TinyNetworking/URLEncoding.swift]: Sounds good, let me change it and push a new commit :)

Comment (asteiman) [reply Sources/TinyNetworking/URLEncoding.swift]: @jdisho pushed the changes

**Review by jdisho — APPROVED** [review]
🚀

**Pull request merged**